.github/workflows/ci.yaml

```
@@ -34,6 +34,13 @@ jobs:
           # Optional: version of golangci-lint to use in form of v1.2 or v1.2.3 or `latest` to use the latest version
           version: v1.52
           working-directory: Proxy
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v3
+        with:
+          # Optional: version of golangci-lint to use in form of v1.2 or v1.2.3 or `latest` to use the latest version
+          version: v1.52
+          working-directory: Proxy
+          args: --timeout 5m
```

```suggestion
          args: --timeout 5m
```
you just need to modify the existing action step, so it runs a little longer